src/databaseUtils.ts

```diff
@@ -31,5 +31,3 @@ export type ViewNames = keyof Views;
 export type ParcelsPlusRow = Views["parcels_plus"]["Row"];
 
 export type ParcelStatus = Schema["status_order"]["event_name"];
-
-export type AuditLogPlusDbRow = ViewSchema["audit_log_plus"];
```

Comment:
I decided to move this out and define the Db Data types in the types file for each server side table. I think this is better because this file will be less cluttered and the types relating to the specific tables will be defined in the types file for that table which I think makes more sense

Reply:
I actually think `AuditLogPlusDbRow` can stay in this file. I know that parcels_plus stuff happens to be used in just the parcels page, but at the end of the day, `AuditLogPlusDbRow` is a db type, and it doesn't make sense to define this in a file that primarily deals with front-end stuff.

Reply:
Okay, I've moved all of the DbRow type definitions in here